src/source/worker_tile.js

```diff
@@ -196,7 +198,10 @@ class WorkerTile {
                     featureIndex,
                     collisionBoxArray: this.collisionBoxArray,
                     glyphAtlasImage: glyphAtlas.image,
-                    imageAtlas: imageAtlas
+                    imageAtlas: imageAtlas,
+                    glyphMap: this.returnDependencies ? glyphMap : null,
+                    iconMap: this.returnDependencies ? iconMap : null,
+                    glyphAtlas: this.returnDependencies ? glyphAtlas : null
```

Comment:
Does it make sense to just pass through `glyphAtlas.positions` to reduce the amount that's transferred between threads?

Reply:
🤔 That's probably a good idea since the positions are all that's needed, although none of this actually transfers between threads in the bench case and `returnDependencies` shouldn't be true in the "live" case.

Reply:
Made the change you suggested.